### PR TITLE
install/kubernetes: Remove manifest sha from image tag for certgen

### PR DESCRIPTION
**Pull request by gandro**
The switch to quay.io as the default docker registry made the `certgen`
image tag invalid, as images built on quay.io and docker.io have a
different SHA (they are built independently on both Quay and DockerHub).

---

Note: Ideally, we would of course have the same manifest SHA for both registries, but that's a more involved change. To fix the getting started guide ASAP and allow people to switch registries, not hard-coding the manifest sha as we do for all other images anyways should do the trick for now.

### CODE REVIEW

File: install/kubernetes/Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ HUBBLE_UI_VERSION := "v0.7.3"
 MANAGED_ETCD_VERSION := "v2.0.7"
 ETCD_VERSION := "v3.4.13"
 NODEINIT_VERSION := "62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8"
```
Comment (sayboras) [inline install/kubernetes/Makefile:10]: not related to this change, will the same issue might happen for start-script and other images ?

**Review by sayboras — APPROVED** [review]
LGTM.

One question as below (not related to this change though), I think the best is still `build once, push everywhere`, but I am not sure how hard it is 😢

File: install/kubernetes/Makefile (hunk 1 above)
Comment (gandro) [reply install/kubernetes/Makefile:10]: I think this node-init image tag refers to the underlying git commit, not the docker manifest sha. So it does not seem affected, *I think*.

**Review by rolinh — APPROVED** [review]

File: install/kubernetes/Makefile (hunk 1 above)
Comment (sayboras) [reply install/kubernetes/Makefile:10]: oh based on the format, I think you are right. I tried to confirm it, but the repo seems empty (or at least for me) https://github.com/cilium/startup-script

**Review by aanm — CHANGES_REQUESTED** [review]
Can you apply this diff please?
```patch
$ git diff
diff --git a/install/kubernetes/Makefile b/install/kubernetes/Makefile
index d5c2f4476..f2f65ab9c 100644
--- a/install/kubernetes/Makefile
+++ b/install/kubernetes/Makefile
@@ -101,7 +101,7 @@ check-docker-images:
          MANAGED_ETCD_VERSION=$(MANAGED_ETCD_VERSION) \
          ETCD_VERSION=$(ETCD_VERSION) \
          NODEINIT_VERSION=$(NODEINIT_VERSION) \
-         CERTGEN_VERSION=`echo $(CERTGEN_VERSION) | egrep -o '^.*@' | sed 's/@//'` \
+         CERTGEN_VERSION=$(CERTGEN_VERSION) \
          ../../contrib/release/check-docker-images.sh "v$(VERSION)"
 
 .PHONY: all check-docker-images clean docs experimental-install lint quick-install update-versions
```

**Comment by gandro** [comment]
@aanm Done!

**Review by aanm — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Pull request merged**